src/Config.php

```diff
@@ -1,19 +1,26 @@
 <?php
 namespace Vendasta\Godaddy\V1;
+
 class Config
 {
     const ENVIRONMENT_PARAMS = [
         "PROD" => [
-            'host' => 'godaddy-api-prod.apigateway.co:443',
-            'scope' => 'https://godaddy-api-prod.apigateway.co',
-            'url' => 'https://godaddy-api-prod.apigateway.co',
+            'host' => 'godaddy-api-prod.vendasta-internal.com:443',
+            'scope' => 'https://godaddy-api-prod.vendasta-internal.com',
+            'url' => 'https://godaddy-api-prod.vendasta-internal.com',
             'secure' => true,
         ],
         "DEMO" => [
-            'host' => 'godaddy-api-demo.apigateway.co:443',
-            'scope' => 'https://godaddy-api-demo.apigateway.co',
-            'url' => 'https://godaddy-api-demo.apigateway.co',
+            'host' => 'godaddy-api-demo.vendasta-internal.com:443',
+            'scope' => 'https://godaddy-api-demo.vendasta-internal.com',
+            'url' => 'https://godaddy-api-demo.vendasta-internal.com',
             'secure' => true,
+        ],
+        "LOCAL" => [
```

Comment:
I don't think we really have a local environment for our service. At least not one that anyone can hit over the network